components/infobox/wikis/starcraft2/infobox_show_custom.lua

```diff
@@ -45,10 +43,6 @@ function CustomInjector:addCustomCells(widgets)
 		content = {CustomShow:_getReleasePeriod(_args.sdate, _args.edate)}
 	})
 
-	if Namespace.isMain() and _args.edate == nil then
-		_show.infobox:categories('Active Shows')
-	end
```

Comment:
is it intended that the `Namespace.isMain()` gets removed?

Reply:
Namespace check is still present in base infobox show